pages/home/mcp-desktop-clients/vscode-client.mdx

```diff
@@ -0,0 +1,45 @@
+import { Steps } from "nextra/components";
+
+# Use Arcade in Visual Studio Code
+
+In this guide, you'll learn how to connect Visual Studio Code to Arcade.dev's MCP server.
+
+<Info>
+As of version 1.100.0, Visual Studio Code does not yet support MCP authorization. For now, only tools that do not require auth, such as math and [search](/toolkits/search/google_search) tools, will work with Visual Studio Code. Stay tuned!
+</Info>
+
+<Steps>
+
+
+### Set up Visual Studio Code
+
+1. Download and open [Visual Studio Code](https://code.visualstudio.com/download) (version 1.100.0 or higher)
+1. Open the command palette and select **MCP: Add Server...**
+1. Choose **HTTP**
+1. Paste the following URL: `https://api.arcade.dev/v1/mcps/ms_0ujssxh0cECutqzMgbtXSGnjorm/mcp`
```

Comment:
This secret is supposed to be here, yes?  One day this will be per-user?

Reply:
Not a secret; an ID. Soon any customer will be able to create one of these.